requirements.txt

```diff
@@ -30,3 +30,7 @@ pydocstyle
 flake8
 vtki
 pylsl>=1.12
+bqplot
+ipyvolume
+pythreejs
+ipywidgets>=7.4
```

Comment:
Also should be added to `environment.yml`

Reply:
I think it's already the case right?

Reply:
It seems to be in `environment.yml`

Reply:
Argh yes, must have missed it or been looking at an old commit